tests/test_models.py

```diff
@@ -181,10 +181,10 @@ def test_timdex_record_all_fields_and_subfields(timdex_record_all_fields_and_sub
         timdex_record_all_fields_and_subfields.locations[0].kind
         == "Data was gathered here"
     )
-    assert timdex_record_all_fields_and_subfields.locations[0].geodata == [
-        -77.025955,
-        38.942142,
-    ]
+    assert (
+        timdex_record_all_fields_and_subfields.locations[0].geoshape
+        == "BBOX(-77.025955, 38.942142)"
```

Comment:
Space between `"BBOX"` and opening parentheses `(`?

Reply:
Sorry, I discovered that it would still index without the space, I had shared that with Graham on Slack but I should have noted it on the PR as well!

Reply:
Ah, good to know! :) Thank you for clarifying.